lib/elixir/lib/list.ex

```diff
@@ -621,7 +621,19 @@ defmodule List do
        :unicode.characters_to_binary(list)
     rescue
       ArgumentError ->
-        raise ArgumentError, "cannot convert list to string. The list must contain only integers, strings or nested such lists; got: #{inspect list}"
+        raise ArgumentError, """
+        Cannot convert this list to a string.
```

Comment:
@josevalim Thank you. Happy to change that. Out of curiosity, is it "just" a matter of consistency or is there also some practical reason behind this convention?